### PR TITLE
Add link to ContentDB profile

### PR DESCRIPTION
**Pull request by rubenwardy**
I haven't tested as I don't have phpBB available

### CODE REVIEW

File: template/memberlist_view.html [hunk 1]
```diff
@@ -36,6 +36,7 @@ <h2 class="memberlist-title">{PAGE_TITLE}</h2>
 		<!-- IF S_USER_INACTIVE --><dt>{L_USER_IS_INACTIVE}{L_COLON}</dt> <dd>{USER_INACTIVE_REASON}</dd><!-- ENDIF -->
 		<!-- IF AGE !== '' --><dt>{L_AGE}{L_COLON}</dt> <dd>{AGE}</dd><!-- ENDIF -->
 		<!-- IF S_GROUP_OPTIONS --><dt>{L_USERGROUPS}{L_COLON}</dt> <dd><select name="g">{S_GROUP_OPTIONS}</select> <input type="submit" name="submit" value="{L_GO}" class="button2" /></dd><!-- ENDIF -->
+		<dt>ContentDB:</dt> <dd><a href="https://content.minetest.net/user/forum/{USERNAME}/">{USERNAME}</a></dd>
```
Comment (sfan5) [inline template/memberlist_view.html:39]: target=_blank ?

Comment (wsor4035) [reply template/memberlist_view.html:39]: > target=_blank ?

aka open in new tab

**Comment by celeron55** [comment]
I did these branch changes in the repo:
- copied master -> phpbb3_1
- copied phpbb3_2 -> phpbb3_3
- added a commit to phpbb3_3 (to fix breadcrumbs)
- force-pushed master to match phpbb3_3
- deleted phpbb3_3 because it doesn't make sense to have two branches doing essentially the same thing in this repo.

In any case due to this I went ahead and retargeted this PR to master. It probably should be rebased.

**Comment by rubenwardy** [comment]
closing as no interest

**Pull request closed without merging**